README.md

```diff
@@ -52,9 +52,10 @@ This repository contains all of the code that runs on [worldcubeassociation.org]
   Feel free to take a look at our [chef recipe](https://github.com/thewca/worldcubeassociation.org/blob/master/chef/site-cookbooks/wca/recipes/default.rb#L6-L23)
   for the accurate versions we use and how we install them.
   Please note that other versions may work, but it is not guaranteed.
-  3. `bundle install && bin/yarn`
-  4. `bin/rake db:load:development` - Download and import the [developer's database export](https://github.com/thewca/worldcubeassociation.org/wiki/Developer-database-export). Depending on your computer it may take a long time. Alternatively you can run `bin/rake db:reset` which will create the database and seed it with random data (it's way faster, but less representative of our website content).
-  5. `bin/rails server` - Run rails. The server will be accessible at localhost:3000
+  3. Install `libyaml` using your package manager of choice, eg. [Mac](https://formulae.brew.sh/formula/libyaml) or [Ubuntu](https://launchpad.net/ubuntu/+source/libyaml)
```

Comment:
In the recipe+Dockerfile below, you install `libyaml-dev`. But here in the Readme, you recommend `libyaml`. I think that these are two different packages.

Reply:
So libyaml-dev are only header files, which are required for psych, in brew you can't actually install the header files only, which is why you install the whole package. For Ubuntu users I link the whole package and the user can choose to install 
the whole package or just the header files.